config/locales/de/models.yml

```diff
@@ -41,25 +41,25 @@ de:
         other: "Veranstaltungsorte"
       event:
         # Basic pluralization
-        single: "Event"
-        plural: "Ereignisse"
+        single: "Veranstaltug"
+        plural: "Veranstaltungen"
         # Countable
-        one: "Event"
-        other: "Ereignisse"
-      online_event: # TODO: Translate
+        one: "Veranstaltung"
+        other: "Veranstaltungen"
+      online_event:
         # Basic pluralization
-        single: "Online Event"
-        plural: "Online Events"
+        single: "Online Veranstaltung"
+        plural: "Online Veranstaltungen"
         # Countable
-        one: "online event"
-        other: "online events"
-      offline_event: # TODO: Translate
+        one: "Online Veranstaltung"
+        other: "Online Veranstaltungen"
+      offline_event:
         # Basic pluralization
-        single: "In-Person Event"
-        plural: "In-Person Events"
+        single: "Vorort Veranstaltung"
+        plural: "Vorort Veranstaltungen"
         # Countable
-        one: "in-person event"
-        other: "in-person events"
+        one: "Vorort Veranstaltung"
+        other: "Vorort Veranstaltungen"
```

Comment:
These should probably not be capitalised, as they will be used in the middle of a sentence.

Reply:
ok. will change to "Vor-Ort-Veranstaltung" as that is what duden suggests.